managed/CounterStrikeSharp.API/Core/Model/CFixedBitVecBase.cs

```diff
@@ -0,0 +1,64 @@
+﻿/*
+ *  This file is part of CounterStrikeSharp.
+ *  CounterStrikeSharp is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  CounterStrikeSharp is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with CounterStrikeSharp.  If not, see <https://www.gnu.org/licenses/>. *
+ */
+
+using System.Runtime.InteropServices;
+
+namespace CounterStrikeSharp.API.Core
+{
+    // credits: qstage
+    [StructLayout(LayoutKind.Sequential)]
+    public unsafe struct CFixedBitVecBase
+    {
+        private const int LOG2_BITS_PER_INT = 5;
+        private const int MAX_EDICT_BITS = 14;
```

Comment:
Might be able to reuse the values from utilities: https://github.com/roflmuffin/CounterStrikeSharp/blob/42dd270b7852c7d866a8bdf835524fc0bd27a787/managed/CounterStrikeSharp.API/Utilities.cs#L35

Reply:
yes we can re use the values!